src/glview/preview/OpenCSGRenderer.cc

```diff
@@ -261,12 +265,16 @@ void OpenCSGRenderer::createCSGVBOProducts(
                                         OpenSCADOperator::DIFFERENCE);
 
         ColorMode colormode = ColorMode::NONE;
+        bool override_color;
         if (highlight_mode) {
           colormode = ColorMode::HIGHLIGHT;
+          override_color = true;
         } else if (background_mode) {
           colormode = ColorMode::BACKGROUND;
+          override_color = true;
         } else {
           colormode = ColorMode::CUTOUT;
+          override_color = true;//c.isValid();
```

Comment:
What is the purpose of leaving commented out code here?

Reply:
Removed